### PR TITLE
Task/6944

### PR DESCRIPTION
**Pull request by shengyuan**
## Description

[redmine issue](https://redmine.kingkit.codes/issues/6944)

## Motivation and Context

~~先上一版，細節再調整。~~
補上語音輸入。

## Screenshots (if appropriate)

https://github.com/webduinoio/pluto-frontend/assets/4637394/1271c25f-3871-4a89-b423-c3f2b6eaf35c

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/hooks/useMqtt.ts [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  pubTopic: string;
+  respTopic_cb: string;
+  respTopic_end: string;
+  onConnectPromise: any;
+  subscriptions: any;
+  failure: boolean;
+
+  constructor(userId: string) {
+    this.userId = userId;
+    this.client = new Paho.Client('wss://mqtt1.webduino.io/mqtt', userId);
```
Comment (GoGoPenguin) [inline src/hooks/useMqtt.ts:14]: 話說，現在的 mqtt server 分別是 `mqtt.webduino.io/mqtt` 跟 `mqtt-ins.webduino.io/mqtt`
`mqtt1` 跟 `mqtt3` 都是指向 `mqtt.webduino.io/mqtt
所以之後就只要打 `mqtt.webduino.io/mqtt` 就好了

File: src/router/index.ts [hunk 2]
```diff
@@ -18,6 +18,29 @@ const router = createRouter({
...
         },
+        {
+          path: 'qa',
+          name: 'StudyBuddyQA',
+          component: () => import('@/views/StudyBuddyQAView.vue'),
+        },
+      ],
+    },
+    {
+      path: '/test',
```
Comment (GoGoPenguin) [inline src/router/index.ts:29]: 這個路由是要做什麼的呢？

File: src/hooks/useMqtt.ts (hunk 1 above)
Comment (shengyuan) [reply src/hooks/useMqtt.ts]: 已調整。

File: src/router/index.ts (hunk 2 above)
Comment (shengyuan) [reply src/router/index.ts]: 測試元件功能用的，調整為 `_test`，並加上 nginx 設定，不過不是很確定是否有效或正確，再麻煩確認。

**Review by GoGoPenguin — APPROVED** [review]

**Pull request merged**